core/templates/dev/head/exploration_editor/main_tab/StateResponses.js

```diff
@@ -306,6 +315,58 @@ oppia.controller('StateResponses', [
         return choiceIndices.every(function(choiceIndex) {
           return handledAnswersArray.indexOf(choiceIndex) !== -1;
         });
+      } else if (interactionId === 'ItemSelectionInput') {
+        var maxSelectionCount = (
+            customizationArgs.maxAllowableSelectionCount.value);
+        if (maxSelectionCount === 1) {
+          var numChoices = $scope.getAnswerChoices().length;
+          // Use array to store booleans that become true
+          // if answerChoice is covered
+          handledAnswersArray = [];
+          for (var i = 0; i < numChoices; i++) {
+            handledAnswersArray.push(false);
+            answerChoices.push($scope.getAnswerChoices()[i].val);
+          }
+
+          var newHandledAnswers = function(current) {
```

Comment:
I think handleAllAnswersExceptGivenOne might be a better name, assuming you aren't removing this function altogether (see later comments). "Other" is a bit vague.